src/ape/managers/query.py

```diff
@@ -21,39 +21,25 @@ class DefaultQueryProvider(QueryAPI):
     Allows for the query of blockchain data using connected provider
     """
 
+    @singledispatchmethod
     def estimate_query(self, query: QueryType) -> Optional[int]:
-        """
-        Estimates the time that the query will take as a timestamp
-
-        Args:
-            query (``QueryType``): The transaction data you want to query
-
-        Returns:
-             Optional[int]: Depends on whether the query can be completed
-        """
```

Comment:
We removed the docstrings per https://github.com/ApeWorX/ape/pull/535#discussion_r820031929

@unparalleled-js do you have any thoughts here in terms of docs rendering?

Reply:
Also note that this class will eventually move to a 1st party plugin

Reply:
Yes, these query subclasses **do not** need documentation unless they wildly differ from the API